frontend/src/pages/_events/slug/default/EventTimeline/index.js

```diff
@@ -51,20 +59,27 @@ const ColorlibConnector = withStyles({
         },
     },
     line: {
-        borderColor: '#19DDEA',
+        borderColor: props => props.accent || '#19DDEA',
 
         borderRadius: 1,
     },
     lineVertical: {
-        borderColor: '#19DDEA',
+        borderColor: props => props.accent || '#19DDEA',
         padding: 0,
 
         borderRadius: 1,
     },
-})(StepConnector)
+})
+
+const ColorlibConnector = withStyles(colorLibStyle())(StepConnector)
+
+ColorlibConnector.propTypes = {
+    ...ColorlibConnector.propTypes,
+    accent: PropTypes.string,
```

Comment:
This makes React happy about the additional prop. I couldn't find a nicer way that works together with `withStyles`